backend/tests/test_stub.py

```diff
@@ -51,11 +58,17 @@ def test_test_endpoint_invalid_input():
 
 @pytest.mark.asyncio
 async def test_books_add_endpoint(book_dict):
+    await TitleMetadata.objects.delete(each=True)
```

Comment:
You should filter on the ident that you'll be creating so that you only delete here what you will create in this test.

By keeping tests isolated, we're saving hard-to-debug issues that could arise as the codebase grow like test unknowingly depending on others and thus randomly failing.